iFixit/react-commerce
# Add collection product list view

This PR introduces product list view in collection

> Note: I've also applied some suggested changes from #4 

## QA

1. Visit the Vercel preview
2. Visit one collection that contains products
3. Verify that products are displayed in a list
4. Verify that you can switch between the list view and the grid view using the buttons above the products list
